openapis/swagger.yaml

```diff
@@ -245,6 +245,37 @@ paths:
       security:
         - basic_auth: []
   '/index/blank/{GUID}':
+    post:
+      tags:
+        - index
+      summary: Create a new, blank version for the document associated to the provided uuid.
+      description: Creates a new blank version of a record with the provided GUID. Returns the GUID of the new version of the record and the baseid common to all versions of the record. Authorization (acl/authz fields) carry over from the original record to the new blank version. No other metadata (md5sum, etc) carries over from the original record to the new blank version.
+      operationId: addNewBlankVersion
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: GUID
+          in: path
+          description: the uuid associated to the record needed to have new version
+          required: true
+          type: string
+        - in: body
+          name: body
+          description: Metadata object that needs to be added to the store
+          required: true
+          schema:
+            $ref: '#/definitions/InputBlankInfo'
+      responses:
+        '200':
```

Comment:
your endpoint returns 201 not 200

and looks like `POST /index/blank` is returning 201 too and the swagger says 200, if you can fix that as well that'd be appreciated 🙏

Reply:
Good catch, thank you! I'll update `/index/blank` as well.